### ISSUE TITLE
INVALID = D and VALID_END_DATE = '12/31/2099' 

### ISSUE THREAD

**Issue by ericaVoss** [OHDSI/Vocabulary-v5.0#29]
**VERSION v5.0 21-Mar-2015**

I noticed 874 CONCEPT_RELATIONSHIPs that are invalid but have a 31-DEC-99 end date.  

```{sql}
SELECT 'CONCEPT_RELATIONSHIP'			AS VOCAB_TABLE,
CAST(CONCEPT_ID_1 AS VARCHAR(50))		AS ID,
'CONCEPT_ID_1'							AS ID_NAME,
'CONCEPT_ID_2 <> RELATIONSHIP_ID <> VALID_START_DATE <> VALID_END_DATE <> INVALID_REASON' AS EVIDENCE_COLUMNS,
CAST(CONCEPT_ID_2 AS VARCHAR(50)) + ' <> ' +
CAST(RELATIONSHIP_ID AS VARCHAR(50)) + ' <> ' + 
CAST(VALID_START_DATE AS VARCHAR(50)) + ' <> ' +
CAST(VALID_END_DATE AS VARCHAR(50)) + ' <> ' +
ISNULL(CAST(INVALID_REASON AS VARCHAR(256)),'') AS EVIDENCE	
FROM CONCEPT_RELATIONSHIP cr
WHERE cr.VALID_END_DATE = '12/31/2099' 
AND cr.INVALID_REASON IS NOT NULL
```

**Comment by cgreich** [comment]
31-Dec-1999 snuck in there somehow. Fixed.

**Comment by ericaVoss** [comment]
Please close this issue, it is fixed in **v5.0 3-Apr-2015**
